include/fizzy/fizzy.h

```diff
@@ -50,11 +50,31 @@ typedef struct FizzyExecutionResult
 typedef FizzyExecutionResult (*FizzyExternalFn)(void* context, FizzyInstance* instance,
     const union FizzyValue* args, size_t args_size, int depth);
 
+/// Value type.
+typedef uint8_t FizzyValueType;
+static const FizzyValueType FizzyValueTypeI32 = 0x7f;
+static const FizzyValueType FizzyValueTypeI64 = 0x7e;
+static const FizzyValueType FizzyValueTypeF32 = 0x7d;
+static const FizzyValueType FizzyValueTypeF64 = 0x7c;
+/// Special value, can be used only as function outuput type.
+static const FizzyValueType FizzyValueTypeVoid = 0;
```

Comment:
Defining it as `0` allows user to create void type with not params as `{}`, at least in C++, probably in C, too.